Atrtistep/Artistep/Controller/Page1MainController.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  ViewController.swift
+//  Atrtistep
+//
+//  Created by Artistep on 2022/07/20.
+//
+
+import UIKit
+
+class Page1MainController: UIViewController {
+
+    //MARK: IBOutlet properties
+    
+    @IBOutlet weak var tableView: UITableView!
+    
+    //MARK: Properties
+    
+    
+    //MARK: Functions
```

Comment:
```suggestion
    //MARK: Life Cycle
```

Life Cycle 관련 함수로 마크해주면 좋을 것 같아요